uk.co.compucorp.civicrm.tasksassignments/js/test/crm-tasks-workflows/controllers/casetype.extended.controller.spec.js

```diff
@@ -111,6 +119,44 @@
           });
         }
       });
+
+      describe('default case type category value', function () {
+        var CATEGORY_FIELD = 'custom_' + CASE_TYPE_CATEGORY_FIELD_ID;
+
+        describe('when creating a new case type', function () {
+          beforeEach(function () {
+            var caseType = getMockCaseType();
+            $scope = $rootScope.$new();
```

Comment:
Lets move this into the beforeEach.

Reply:
move what specifically? isn't the `$scope` already inside the beforeEach?